docs/CTCORECHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 ## CleverTap Android SDK CHANGE LOG
 
+### Version 4.6.6 (October 31, 2022)
+* Fix app inbox bug where inbox message videos would not play when new inbox notifications are available
```

Comment:
@root-ansh-ct Change this to - "Fixes App Inbox bug where an Inbox message's video would not play when new Inbox messages were available"